app/appshell/app.js

```diff
@@ -210,8 +211,8 @@ app.getRemoteDebuggingPort = function () {
 };
 
 app.getUserDocumentsDirectory = function () {
-    // TODO: implement
-    throw new Error("app.getUserDocumentsDirectory not implemented");
+    var _path = path.resolve(electronApp.getPath("home"), "Documents");
```

Comment:
Just looked it up in Linux Mint.
It exists, but unfortunately, it's named `Dokumente` on my German OS. So `/home/marcel/Documents` _doesn't_ exist.